src/metrics/prometheus_counter.erl

```diff
@@ -90,7 +90,7 @@ inc(Caller) ->
 -define(TABLE, ?PROMETHEUS_COUNTER_TABLE).
 -define(ISUM_POS, 2).
 -define(FSUM_POS, 3).
--define(WIDTH, 16).
+-define(WIDTH, erlang:system_info(schedulers_online)).
```

Comment:
Maybe instead of fixing this to be `schedulers_online` we can set this up as an `application:get_env(<application>, <some_name>, 16)` so the original remains preserved but we open up the loophole for setting a higher value.

Reply:
Another issue is that `schedulers_online` can change value dynamically at runtime.

Reply:
Right. Should use just `schedulers`, as that can't change at runtime.

On the other hand I'm not sure it'd help to make it configurable, currently the algorithm that chooses the key would do `X band (?WIDTH - 1)`, so most values would mean skewing the distribution of schedulers to each key and some keys would see more contention than others. Sensibly, if we're keying on the scheduler id, the only values that make sense are multiples (or halves) of the number of schedulers.